src/styles/app.styl

```diff
@@ -99,13 +99,18 @@ header
         box-shadow      none
 
 #maif-figure, #fing-figure
-    .svg-wrapper
-        background-color $maif
-
     svg
         max-width       100%
```

Comment:
Yup still do. I added that because the default style wasn't what i needed. But i agree that some refactoring wouldn't be useless. I hate working with #ids in CSS 😠